modules/stochastic_tools/src/distributions/NormalDistribution.C

```diff
@@ -0,0 +1,32 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+#include "NormalDistribution.h"
+
+template <>
+InputParameters
+validParams<NormalDistribution>()
+{
+  InputParameters params = validParams<Distribution>();
+  params.addClassDescription("Boost Normal distribution.");
+  params.addRequiredParam<Real>("location", "The Normal location parameter.");
+  params.addRequiredParam<Real>("scale", "The Normal scale parameter.");
```

Comment:
changes these to be 'standard_deviation' and 'mean', make sure it matches what is passed to boost.